docs/plugin.go

```diff
@@ -37,74 +37,74 @@ const (
 	PluginCompileUse   string = `compile [compile options...] <host_path>`
 	PluginCompileShort string = `Compile a singularity plugin`
 	PluginCompileLong  string = `
-	The 'plugin compile' command allows a developer to compile a singularity plugin in the
-	expected environment. The provided host directory is the location of the plugin's
-	source code.`
+The 'plugin compile' command allows a developer to compile a singularity plugin 
+in the expected environment. The provided host directory is the location of the 
+plugin's source code.`
```

Comment:
Can you put the 2 spaces here? To be consistent with all the other helps:

```suggestion
  plugin's source code.`
```